spec/acceptance/spec_helper.js

```diff
@@ -0,0 +1,28 @@
+var mocha = require('mocha'),
+		chai = require('chai'),
+		chaiAsPromised = require('chai-as-promised'),
+		nodemon = require('nodemon'),
+		expect = chai.expect,
+		env = require('./environment'),
+		child_process = require('child_process'),
+		fs = require('fs'),
+		app = require('../../server'),
+		debug = require('debug')('gokibitz'),
+		io = require('../../server/io'),
+		server;
+
+chai.use(chaiAsPromised);
+global.expect = expect;
+
+app.set('port', 3435); // Use a different port from development server
```

Comment:
Good port number: the 3-4 / 3-5 opening is an advanced but good opening. http://eidogo.com/#18Ie8mufo:0,4